src/components/input/DurationInput.js

```diff
@@ -302,9 +302,9 @@ class DurationInput extends PureComponent {
                 <div className={theme['input-inner-wrapper']} style={{ flex: '1 2 auto' }}>
                   <InputBase
                     value={
-                      splitSecondsIntoHoursMinutesSeconds(seconds).minutes < 10
-                        ? `0${splitSecondsIntoHoursMinutesSeconds(seconds).minutes}`
-                        : splitSecondsIntoHoursMinutesSeconds(seconds).minutes
+                      this.splitSecondsIntoHoursMinutesSeconds(seconds).minutes < 10
```

Comment:
You moved it out, but you're still calling it 9 times.

- call it once in above the return
- destructure the result into hours, minutes and seconds
- use the minutes, seconds, hours variables at the places where you call the function